src/test/factories/cycle.js

```diff
@@ -14,5 +18,9 @@ export default function define(factory) {
     state: CYCLE_STATES[0],
     createdAt: cb => cb(null, now),
     updatedAt: cb => cb(null, now),
+    startOfWeek: cb => cb(null, {
+      mmddyy,
+      unixTimestamp,
+    }),
```

Comment:
We've established a (commonly used) convention of having datetime fields in a model stored as a single JS date (as opposed to an object with multiple props, including a date value). We also have a convention of naming these fields `*At` (ex: `createdAt`, `updatedAt`).

We should stick with this convention for a new field for the week starting timestamp and use the property name `weekStartedAt`. It should store only a date value. The formatting should be left to be determined on the client-side.